Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -449,33 +451,47 @@ private SiteExtensionInfo GetPreInstalledExtension(string id)
                 string packageLocalFilePath = GetNuGetPackageFile(package.Identity.Id, package.Identity.Version.ToString());
                 using (tracer.Step("Download site extension: {0}", package.Identity))
                 {
-                    await remoteRepo.DownloadPackageToFolder(package.Identity, installationDirectory, pathToLocalCopyOfNudpk: packageLocalFilePath);
-                }
+                    string extractPath = installationDirectory;
+                    if (SiteExtensionInfo.SiteExtensionType.WebRoot == type)
```

Comment:
Not a big deal (and no need to change), but we don't usually use reversed comparisons in C#. These were useful in C/C++ where they helped catch cases where `=` was used by mistake instead of `==`. But C# prevents this.